format/BUILD.bazel

```diff
@@ -34,6 +34,44 @@ java_binary(
     runtime_deps = ["@google-java-format//jar"],
 )
 
+java_binary(
+    name = "scalafmt_cli_2_13",
+    main_class = "org.scalafmt.cli.Cli",
+    runtime_deps = [
```

Comment:
nit: any comment you can add here about maintaining this list? Ideally a `bazel query` one-liner that can be run to reproduce it, in case it changes when we upgrade scalafmt someday?